src/components/Footer/index.js

```diff
@@ -0,0 +1 @@
+export { default as Footer } from './Footer.jsx';
```

Comment:
You should add "resolve" rule to the webpack config, and don't write '.jsx' extension